torch/_inductor/config.py

```diff
@@ -100,6 +100,11 @@ def is_fbcode():
         else os.cpu_count(),
     )
 )
+# be able to override compile_threads. We may want to disable async compiling by
```

Comment:
Merge this with the above line (perhaps a helper function)? A bit confusing to set this variable twice.